### PR TITLE
Updated html and css

### PR DESCRIPTION
**Pull request by IvanTkachenko3301**
[DEMO LINK](https://IvanTkachenko3301.github.io/layout_html-form/)
[TEST REPORT LINK](https://IvanTkachenko3301.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,36 @@
...
+
+.last-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.label-last-form-bottom {
+  display: block;
+  margin-bottom: 10px;
+}
```
Comment (liliia-livandovskaya) [inline src/style.css]: Rewrite styles and classes. For now, you have a lot of duplicated styles. Try to improve it

File: src/index.html [hunk 2]
```diff
@@ -3,15 +3,180 @@
...
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="last-form">
+        <legend>Additional info</legend>
+
+        <label class="comments label-last-form-bottom">
+          Comments:
```
Comment (liliia-livandovskaya) [inline src/index.html:160]: checklist p.8 - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.

Screenshot 2023-02-20 at 16 13 47

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! 🔥 Pay attention to the comments

File: src/style.css (hunk 1 above)
Comment (IvanTkachenko3301) [reply src/style.css]: Done

File: src/index.html (hunk 2 above)
Comment (IvanTkachenko3301) [reply src/index.html:160]: Done

**Pull request closed without merging**